README.md

```diff
@@ -51,22 +51,38 @@ Windows:
 bin\hunter_env.cmd
 ```
 
-After the environment is configured, you can build for any supported `Polly` toolchain (see `polly.py --help`) using the `bin/drishti_build.{sh,cmd}` scripts.
+After the environment is configured, you can build for any supported `Polly` toolchain (see `polly.py --help`) with a command like this:
 
-Linux/OS X:
 ```
-TOOLCHAIN=osx-10-11-hid-sections-lto
-CONFIG=Release  
-INSTALL=--install
-bin/drishti_build.sh "${TOOLCHAIN}" "${CONFIG}" "${INSTALL}"
+polly.py --toolchain ${TOOLCHAIN} --config ${CONFIG} --fwd HUNTER_CONFIGURATION_TYPES=${CONFIG} --install --verbose
 ```
 
+You can override the default CMake options in the project by adding KEY=VALUE pairs to a forwarded list like this `--fwd DRISHTI_SOME_OPTION1=YES DRISHTI_SOME_OPTION2=YES`.  The `--fwd HUNTER_CONFIGURATION_TYPES=${CONFIG}` option is specified above to limit the Hunter package builds to a single configuration type (typically Release) when using multi-configuration generators (Visual Studio, Xcode, etc).  It is not required for standard `clang` or `gcc` builds.
+
+The configurations listed below have all been tested.  In general, most C++11 toolchains should work with minimal effort.  A CI comment indicates that the configuration is part of the Travis or Appveyor CI tests, so all Hunter packages will be available in the server side binary cache.
+
+Linux (Ubunty Trusty 14.04):
```

Comment:
I guess we can save some space by omitting `TOOLCHAIN=`. Just say that "Next arguments for --toolchain tested and can be used...".

Reply:
I could go either way on this.  The `CONFIG` does impact the cache.  If a user wants a quick build they will need to know which `TOOLCHAIN` + `CONFIG` pair to use.